packages/fxa-admin-panel/src/components/Nav/index.tsx

```diff
@@ -4,96 +4,103 @@
 
 import React from 'react';
 import { NavLink } from 'react-router-dom';
-import mailIcon from '../../images/icon-mail.svg';
+import accountIcon from '../../images/icon-account.svg';
+import keyIcon from '../../images/icon-key.svg';
 import statusIcon from '../../images/icon-site-status.svg';
 import logsIcon from '../../images/icon-logs.svg';
 import { AdminPanelFeature } from 'fxa-shared/guards';
 import Guard from '../Guard';
 
-export const Nav = () => (
-  <nav className="mb-4 desktop:mr-5 desktop:flex-1 desktop:mb-0">
-    <div className="p-4 rounded-md bg-white border border-grey-100">
-      <h2 className="mb-3 uppercase text-sm tracking-wide font-normal text-grey-500">
-        Navigation
-      </h2>
-      <ul className="list-none p-0 m-0 text-sm">
-        <Guard features={[AdminPanelFeature.AccountSearch]}>
-          <li>
-            <NavLink
-              to="/account-search"
-              className={({ isActive }) =>
-                `rounded text-grey-600 flex mt-2 px-3 py-2 no-underline hover:bg-grey-100 focus:bg-grey-100 ${
-                  isActive ? 'bg-grey-50 font-semibold' : 'bg-grey-10'
-                }`
-              }
-            >
-              <img
-                className="inline-flex mr-2 w-4"
-                src={mailIcon}
-                alt="mail icon"
-              />
-              Account Search
-            </NavLink>
-          </li>
-        </Guard>
-        <Guard features={[AdminPanelFeature.SiteStatus]}>
-          <li>
-            <NavLink
-              to="/site-status"
-              className={({ isActive }) =>
-                `rounded text-grey-600 flex mt-2 px-3 py-2 no-underline hover:bg-grey-100 focus:bg-grey-100 ${
-                  isActive ? 'bg-grey-50 font-semibold' : 'bg-grey-10'
-                }`
-              }
-            >
-              <img
-                className="inline-flex mr-2 w-4"
-                src={statusIcon}
-                alt="status icon"
-              />
-              Site Status
-            </NavLink>
-          </li>
-        </Guard>
-        <Guard features={[AdminPanelFeature.AccountLogs]}>
+export const Nav = () => {
+  const getNavLinkClassName = (isActive: boolean) =>
```

Comment:
Random nit pick. Maybe put this in the outer scope? Seems like adding this function, resulted in a using a 'return' statement, which in turn had a big impact on the the diff.